### DISCUSSION TITLE
vulnerability issue

### DISCUSSION THREAD

**Discussion by dirkckau** [jaegertracing/jaeger-operator#1694] [Q&A]
Hi, 

In the Security scanning, we found that the Jaeger-operator 1.29.1 image version is having some vulnerability issues. 
Especially on a high severity item "RHSA-2020:3247 ", which requires to upgrade package python3-six to version '0:1.12.0-1.el8ost'. 

![image](https://user-images.githubusercontent.com/91056335/147170189-aba71833-c63f-421c-9e0b-4d43c4a71df8.png)

Is there any plan to include this package? 

Thanks!

**Comment by p-rog** [comment]
How that report was generated?

The RHSA-2020:3247 is for the RHV Manager (ovirt-engine) product.
The Jaeger-operator does not use any packages from the RHV Manager. 
Additionally, indeed the RHSA-2020:3247 is an important security advisory for the RHV Manager, but there is no any security fixes (not related CVEs) to the mentioned python3-six package and the security rating is related to the apache-commons security fixes when that package is not included in the Jaeger-operator image at all. See details at:
https://access.redhat.com/errata/RHSA-2020:3247

There is no any relation between RHSA-2020:3247 (released for RHV) and Jaeger Operator.
It seems that tool where that report was generated incorrectly matched the products CPE.
